docs/iot-platform-sdk/running-the-demo.md

```diff
@@ -508,9 +572,9 @@ Given an OwnershipVoucher file with a unique deviceId/GUID, the following needs
 
 The following is a sample Python\* script that, given an Owner voucher file as an argument, extracts the GUID and creates the previously mentioned device directory structure.
 
-This sample script operates under the assumption that both the script and voucher files are placed at \<sdo-iot-platform-sdk-root>/demo/ocs/config/db/v1/devices, and that this is the current working directory:
+This sample script operates under the assumption that both the script and voucher files are placed at <sdo-iot-platform-sdk-root>/demo/ocs/config/db/v1/devices, and that this is the current working directory:
```

Comment:
location to place the script is not correct. please rectify this path.

Reply:
Path points to the voucher extension python script.